dashboard/legacy/middleware/helpers/projects.rb

```diff
@@ -266,7 +266,7 @@ def increment_abuse(channel_id, amount = 10)
     row = @table.where(id: project_id).exclude(state: 'deleted').first
     raise NotFound, "channel `#{channel_id}` not found" unless row
 
-    new_score = row[:abuse_score] + (JSON.parse(row[:value])['frozen'] ? 0 : amount)
+    new_score = row[:abuse_score] + (JSON.parse(row[:value])['frozen'] ? -50 : amount)
```

Comment:
could the magic number be pulled out into a constant somewhere?

`DEFAULT_FROZEN_ABUSE_SCORE` or whatever?